src/ngraph/ngraph_compiler.cc

```diff
@@ -92,17 +92,20 @@ void Compiler::Infer(const BindArg* bind) {
     if (mutable_nodes.count(nid)) {
       shapes_.push_back(bind->aux_states_[aux_top].shape());
       dtypes_.push_back(bind->aux_states_[aux_top].dtype());
+      stypes_.push_back(bind->aux_states_[aux_top].storage_type());
       ++aux_top;
     } else {
       shapes_.push_back(bind->in_args_[arg_top].shape());
       dtypes_.push_back(bind->in_args_[arg_top].dtype());
+      stypes_.push_back(bind->in_args_[arg_top].storage_type());
       ++arg_top;
     }
   }
 
   // append default shapes / dtypes so that vector size = graph size
   shapes_.resize(idx.input_nodes().size(), nnvm::TShape());
   dtypes_.resize(idx.input_nodes().size(), -1);
+  stypes_.resize(idx.input_nodes().size(), mxnet::kDefaultStorage);
```

Comment:
I was unsure when I was working on this... should it be Default or should it be Undefined?

Reply:
nothing of sparse-type comes inside ngraph-bridge, we initialize to default value (kDefaultStorage) based on comment above. Undefined should be ok too, as i expect this to get initialized to kdefaultstorage at any point.

Reply:
I think either would work but I am fine with default.